contracts/CrossChain.sol

```diff
@@ -322,9 +398,141 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
         registeredContractChannelMap[handlerContract][channelId] = isEnable;
         emit enableOrDisableChannel(channelId, isEnable);
       }
+    } else if (Memory.compareStrings(key, "emergencySuspendApproveThreshold")) {
```

Comment:
Please refactor the name of these parameters.

Reply:
fixed